vertx-web/src/main/java/io/vertx/ext/web/handler/SessionHandler.java

```diff
@@ -239,4 +261,12 @@ static SessionHandler create(SessionStore sessionStore) {
    */
   @Fluent
   SessionHandler setCookieless(boolean cookieless);
+
+  /**
+   * Create a new session
+   *
+   * @param context the routing context
+   * @return the session
+   */
+  Session newSession(RoutingContext context);
```

Comment:
From a quick look to the PR this seems to be used only on a test? If this is a test helper, probably we should not add new methods to the API

Reply:
This is used for session regeneration while a sockjs connection is still open. It's intended to be used outside of tests as well.